test/test_chrome_middleware_node.js

```
@@ -0,0 +1,242 @@
+#!/usr/bin/env node
+
+// Node.js test script for Chrome middleware
+// Tests the structure and logic of the Chrome class
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)